clrdefinitions.cmake

```diff
@@ -177,9 +177,9 @@ if(CLR_CMAKE_PLATFORM_UNIX)
   add_definitions(-DFEATURE_PAL_SXS)
   add_definitions(-DFEATURE_PAL_ANSI)
 endif(CLR_CMAKE_PLATFORM_UNIX)
-if(CLR_CMAKE_PLATFORM_LINUX)
+if(CLR_CMAKE_PLATFORM_LINUX OR CLR_CMAKE_PLATFORM_FREEBSD)
     add_definitions(-DFEATURE_PERFMAP)
```

Comment:
Is this a thing we need for FreeBSD? I ask because we don't seem to have it on macOS...

Reply:
that enables crossgen feature used in core-sdk. That part is currently disabled for OSX & FreeBSD. I bumped to this while investigating crossgened cli.